tests/AdaptiveExpressions.Tests/ExpressionParserTests.cs

```diff
@@ -497,6 +497,11 @@ public class ExpressionParserTests
             Test("bag.name == null ? \"hello\": bag.name", "mybag"),
             Test("one > 0? one : two", 1),
             Test("hello * 5?'r1':'r2'", "r2"),
+            Test("timestampObj < timestampObj2", false),
+            Test("timestampObj > timestampObj2", true),
+            Test("timestampObj >= timestampObj2", true),
+            Test("timestampObj <= timestampObj2", false),
+            Test("timestampObj == timestampObj2", false),
             #endregion
```

Comment:
For each comparison should make sure there is a test that returns true and false.

Reply:
added test cases for the other evaluation for each comparison operator